cerberus/api.py

```diff
@@ -125,31 +124,35 @@ def complete(self, request, pk=None):
         return self.change_state("complete", request)
 
     @action(detail=True, methods=["PUT"])
-    def move_booking(self, request, pk=None):
-        booking = self.get_object()
-        incoming = BookingMoveSerializer(data=request.data)
-        incoming.is_valid()
+    def move(self, request, pk=None):
+        booking: Booking = self.get_object()
+        incoming = ToDateSerializer(data=request.data)
+        if not incoming.is_valid():
+            incoming = ToDateTimeSerializer(data=request.data)
+            incoming.is_valid()
         status = 400
 
         with transaction.atomic(), contextlib.suppress(TransitionNotAllowed):
             with FSMLogDescriptor(booking, "by", request.user):
-                if booking.move_booking(parser.parse(incoming.data["to"])):
+                if booking.move_booking(incoming.validated_data["to"]):
                     status = 200
 
         serializer = self.get_serializer(booking)
 
         return Response({"item": serializer.data, "status": status}, status=status)
 
     @action(detail=True, methods=["PUT"])
-    def move_booking_slot(self, request, pk=None):
-        booking = self.get_object()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure consistency in API endpoint naming.

Renaming 'move_booking_slot' to 'move_slot' improves consistency with the 'move' method for bookings. Verify that all related documentation and frontend calls are updated accordingly.